docs/user-guide/dream/dream-data-reduction.ipynb

```diff
@@ -181,9 +181,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "intermediates[MaskedData[SampleRun]].bins.concat().hist(\n",
-    "    two_theta=300, wavelength=300\n",
-    ").plot(norm=\"log\")"
+    "two_theta = sc.linspace(\"two_theta\", 0.8, 2.4, 301, unit=\"rad\")\n",
```

Comment:
Why the explicit limits? Does this really matter so much for performance?

Reply:
Because we have some voxels with NaN positions, and get an exception otherwise.